.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -0,0 +1,159 @@
+# This is a reusbale workflow to bump the 'sec-scanners-config.'
+#
+# It will do so by using the script `hack/scripts/render-sec-scanners-config.sh`, that is not part of the workflow.
+# If you want to run this workflow against a repo the script must exist in that repo. This is by design, because every repo
+# will require a specfic sec-scanners-config.yaml.
+#
+# To create a PR and monitor it, this workflow will require a classic github personal access token (pat) passed
+# as a secret named `BOT_PAT`. The token must be configured to have all rights for `repo`, `user` and `workflow`.
+# Further reads:
+#   Setting a secret for a repo: https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions
+#
+# If changes were done by the script, the workflow will create a PR and wait for it to be merged.
+# The waiting will happen with a timeout that can be set via the input of `timeout`. The units are seconds.
+# It has a default value of 3600 (seconds (= 1 hour)).
+#
+# Examples of using this workflow:
+#  1. Set all awailable inputs and secrets.
+#
+#   jobs:
+#     call-this-workflow:
+#       uses: kyma-project/eventing-tools/.github/workflows/bump-sec-scanners-config-reusable.yml@main
+#       with:
+#         timeout: 3600 # 1 hour
+#       secrets:
+#         BOT_PAT: ${{ secrets.my_pat }}
+#
+#  2. Minimal setup:
+#
+#   jobs:
+#     call-this-workflow::working_dir:  g
+#       uses: kyma-project/eventing-tools/.github/workflows/bump-sec-scanners-config-reusable.yml@main
+#       secrets:
+#         BOT_PAT: ${{ secrets.my_pat }}
+
+name: Lint code (reusable)
```

Comment:
Any reason for this name?

Reply:
Thanks! No, it is just a typo. I will fix this.